### DISCUSSION TITLE
Welcome to homeassistant-goecharger Discussions!

### DISCUSSION THREAD

**Discussion by cathiele** [cathiele/homeassistant-goecharger#36] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by gitfux1** [comment]
Hello!
While trying to install "HomeAssistant-geocharger" found this in README.rd:
"* setup your Charger in the configuration.yml (for always connected chargers):"
=> should be configuration.yaml ?

Thanks for your well done work ...

**Reply by cathiele** [reply]
Thank you, i fixed it!

**Comment by trekki1990** [comment]
Hi! At first: Thank you for your outstanding work :) I can use your component in my HA installation perfectly. Now I want to try to translate some things into german. I've searched for the status messages in your code (e.g. Charger ready, no vehicle) but can't find these words. Where do you save this status messages? Can I change them? 

Thanks a lot.
Regards

**Comment by NilesDavis** [comment]
Hi! first of all: thanks for this nice integration! could you provide code for the sample view? I just have one switch an all the sensors available, so I have no idea how to set the charge limit or max current. sorry for this newbee question...

Cheers,
Nils

**Reply by NilesDavis** [reply]
alright, I got it working! It was just a seconds restart neccessary for the inputs to work.
Thanks!

**Comment by EvilPutz** [comment]
Hi, thank you for the plugin (: , are there any plans on integrating the new Energy Dashboard ? 
Thanks

**Comment by ratormat** [comment]
Hello, thank you also for the plugin. I was able to load my photovoltaic excess into my car =)
If I understand it right, the set_max_current-service uses the key “amp” on the go eCharger. Works fine and what it should, but for frequent use “amx” is recommended (no flashing). On the GoeCharger class setTmpMaxCurrent() instead of setMaxCurrent().
So my propose is to provide this also in the integration. As separate service or switched by a property.

**Comment by Gromo42** [comment]
Hi, many thanks for this very helpful plugin.
I'm using the latest version of go-e charger with firmware 055.8; hardware version V4.; app version 3.2.11
from the go-e charger documentation I learn there was a change in the name for temperature key: from prior "tmp" to now "tpa"
so the temperature reported from your plugin is always 0 degrees celsius
does anyone know how/where to update this key in the plugin?
Thanks
